src/haxelib/client/Main.hx

```diff
@@ -1037,17 +1048,17 @@ class Main {
 		}
 		else
 		{
-			var inf = try site.infos(p) catch( e : Dynamic ) { Sys.println(e); return; };
-			p = inf.name;
```

Comment:
Oh so there was a reason, thought it was weird to do that if the value was identical.
It a problem though :/ and adding a `realName` function to SiteApi just for that seems suboptimal, would have the merit to be explicit at least. Wouldn't happen to be any other places that do that?